client/src/Components/Pages/Dashboard/News/index.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import newImg from './images/internet.png'
+import paper from './images/news.png'
+
+import "./style.css";
+
+const  index = (props)=>{
+const {news} = props;
+return(
+  <React.Fragment>
```

Comment:
Here's another place where you don't need the React.Fragment.

Reply:
I will modify them in all pages